packages/utilities/src/diff/doc/IWebDiffGenerator.ts

```diff
@@ -0,0 +1,25 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+/**
+ * Web help genrator API that handles genration of web diff base launcher
+ * at cli home dir
+ * @export
+ * @interface IWebDiffGenerator
+ */
+export interface IWebDiffGenerator {
+
+    /**
+     * build the diff generator
+     */
+    buildDiffDir(): void;
+
+}
```

Comment:
Creating the interface is only necessary if we use it elsewhere. Currently this one seems unused unless I have missed where we use it.

If the interface is desired, I would suggest updating the WebDiffGenerator class to implement it (`export class WebDiffGenerator implements IWebDiffGenerator`). Or if it is not needed anywhere, it may be ok to remove it?

Reply:
yeah, I will implement this interface, I missed doing so. :slightly_smiling_face: